.github/workflows/download-data.yml

```diff
@@ -1,37 +1,29 @@
 name: download-data
 
 on:
-    schedule:
-        # Run everyday at 2:00pm CST which is 7:00pm UTC
-        - cron: 0 19 * * *
```

Comment:
If  `cron`  is removed here, is there another place where it will be run?

Reply:
@dcjohnson24 Apologies - I had turned off the cron just as I was working on this, but I forgot to restore it back. I will update this PR - thanks for pointing that out.